modules/UI/videolayout/VideoContainer.js

```diff
@@ -537,10 +534,6 @@ export class VideoContainer extends LargeContainer {
         // flickering effect when scrolling between videos and have the screen
```

Comment:
Does the TO FIX comment here still apply?

Reply:
No. I have removed it.